src/components/buttons/primary-action-button/primary-action-button.js

```diff
@@ -0,0 +1,88 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import classnames from 'classnames';
+import filterDataAttributes from '../../../utils/filter-data-attributes';
+import filterAriaAttributes from '../../../utils/filter-aria-attributes';
+import Spacings from '../../spacings';
+import styles from './primary-action-button.mod.css';
+
+const getStateClassName = ({ isToggled, isDisabled }) => {
+  if (isDisabled) return styles.disabled;
+  if (isToggled) return styles.active;
+  return styles.button;
+};
+const getSizeClassName = size => styles[size];
+
+const PrimaryActionButton = props => {
+  const dataProps = {
+    'data-track-component': 'PrimaryActionButton',
+    ...filterDataAttributes(props),
+  };
+  const ariaProps = {
+    'aria-label': props.ariaLabel,
```

Comment:
Maybe we can do `'aria-label': props.ariaLabel || props.label` just to make sure we won't loose any accessibility with the migration?